src/core/src/pass/visualize_tree.cpp

```diff
@@ -343,12 +343,14 @@ static std::string pretty_partial_shape(
             if (!first) {
                 str << ",";
             }
-            if (const auto& symbol = d.get_symbol()) {
-                const auto& root = ov::symbol::ancestor_of(symbol);
-                if (symbol_map.count(root))
-                    str << "<" << symbol_map.at(root) << ">";
-                else
-                    str << "<?>";
+            if (d.is_dynamic()) {
```

Comment:
Will be any value to print all symbols when building e.g. in `Debug` or some `DEBUG_CAPS` flag?

Reply:
I don't think so, even for the same static dimension they may be different, so I don't think this is necessary.